src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,106 @@
+"use client"
+
+import * as React from "react"
+
+import styled from "@emotion/styled"
+import { default as MuiAlert } from "@mui/material/Alert"
+import type { AlertColor } from "@mui/material/Alert"
+import { Theme } from "@emotion/react"
+
+const getColor = (theme: Theme, severity: AlertColor) => {
+  return {
+    info: theme.custom.colors.blue,
+    success: theme.custom.colors.green,
+    warning: theme.custom.colors.orange,
+    error: theme.custom.colors.lightRed,
+  }[severity]
+}
+
+type AlertStyleProps = {
+  severity: AlertColor
+}
+
+const AlertStyled = styled(MuiAlert)<AlertStyleProps>(
+  ({ theme, severity }) => ({
+    padding: "11px 16px",
+    borderRadius: 4,
+    borderWidth: 2,
+    borderStyle: "solid",
+    borderColor: getColor(theme, severity),
+    background: "#FFF",
+    ".MuiAlert-message": {
+      ...theme.typography.body2,
+      color: theme.custom.colors.darkGray2,
+      alignSelf: "center",
```

Comment:
This `alignSelf` is the only alert change compared to https://mitodl.github.io/mit-learn/?path=/story/smoot-design-alert--basic